R/deprecated.R

```diff
@@ -139,3 +153,92 @@ page_fill <- function(...) {
   .Deprecated("page_fillable", old = "page_fill")
   page_fillable(...)
 }
+
+#  Legacy Nav Containers --------------------------------------------------
+deprecate_if_not_called_from_shiny <- function(old_name, new, version) {
```

Comment:
```suggestion
# This is needed since some shiny functions like `tabPanel()` call
# `nav()` underneath the hood. In a future release of shiny, we 
# could/should require bslib > 0.4.2, update it to use the new version 
# (`nav_panel()`), then we won't need this fancy deprecation 
deprecate_if_not_called_from_shiny <- function(old_name, new, version) {
```